source/features/show-whitespace.tsx

```diff
@@ -17,7 +17,7 @@ const viewportObserver = new IntersectionObserver(changes => {
 	}
 });
 
-function showWhitespaceWhenInViewport(line: HTMLElement): void {
+function showWhitespaceWhenInViewport(line: Element): void {
```

Comment:
@g-plane I think your recent changes somehow broke this. It appeared as soon as I installed the update

Reply:
It shouldn't be, because you're using your own type helper.

Reply:
See https://www.typescriptlang.org/play?target=2#code/JYWwDg9gTgLgBAchgTzAUwCYFoCOBXNKZLAZzQBs0BjGaBAbgChG0APSWOKiAOxPm4Y0AUUog0PGCQDKFarShwAvHADajAJAIAdACNyEXVkFoswHj0IIANHAD0duAGEIQuOXNoSmhAC4eEDAAFNoBMFAAhnzkETBoAJRwAHxwoYGR0bFoNvaOLm76EFQA1iRw5lwQIOKSJNpwALIRyHC6aHAA7pFg6BhwMB3AVGiMALpwEWXcfDBMzNP8cKzKcBhFeDUw2viEyLKUNNBBAAa+wCRBACQA3iaiaJsycodQ2gBWEOZBNgjxAL7xfyBEKFIwmLAACzwPGK8WO8SYcCRDjgAD0APxAA

It's `never` for some reason, likely causes by `.join()` returning a simple `string`

Reply:
You can change the playground version with a comment but it requires a reload to take effect. Here it is working with v2.8

<img width="388" alt="Screenshot" src="https://user-images.githubusercontent.com/1402241/220616062-e20c7cf5-0579-4337-af39-74a4ff30dd03.png">


and not working with v2.9.0

<img width="385" alt="Screenshot 1" src="https://user-images.githubusercontent.com/1402241/220616176-d2c7735e-c7ee-46ba-91ca-a3ca4ca4d90b.png">